tempesta_fw/http_parser.c

```diff
@@ -604,11 +605,49 @@ __FSM_STATE(prefix ## _Done) {						\
 	return TFW_BLOCK;						\
 }
 
-/*
- * Read LWS at arbitrary position and move to stashed state.
- * This is bit complicated (however you can think about this as
- * a plain pushdown automaton), but reduces FSM code size.
+/* In request we should pass empty headers:
+ * RFC 7230 5.4:
+ * ....
+ * ....
+ * If the authority component is missing or
+ * undefined for the target URI, then a client MUST send a Host header
+ * field with an empty field-value.
+ *
+ * NOTE: using of RGEN_LWS_empty should be matched with
+ * the BUG_ON() statements in tfw_http_msg_hdr_val function
  */
+#define RGEN_LWS_common_cases(st)				\
```

Comment:
The macro is just same piece of code for the macros below, so I'd prefer name it ```__RGEN_LWS```